pandas/tests/extension/base/constructors.py

```diff
@@ -33,6 +33,32 @@ def test_series_constructor(self, data):
         assert result2.dtype == data.dtype
         assert isinstance(result2._mgr.blocks[0], ExtensionBlock)
 
+    def test_series_constructor_no_data_with_index(self, data, na_value):
+        dtype = data.dtype
+        result = pd.Series(index=[1, 2, 3], dtype=dtype)
+        expected = pd.Series([na_value] * 3, index=[1, 2, 3], dtype=dtype)
+        self.assert_series_equal(result, expected)
+
+    def test_series_constructor_scalar_na_with_index(self, data, na_value):
+        dtype = data.dtype
+        result = pd.Series(na_value, index=[1, 2, 3], dtype=dtype)
+        expected = pd.Series([na_value] * 3, index=[1, 2, 3], dtype=dtype)
+        self.assert_series_equal(result, expected)
+
+    def test_series_constructor_scalar_with_index(self, data):
+        scalar = data[0]
+        dtype = data.dtype
+        result = pd.Series(scalar, index=[1, 2, 3], dtype=dtype)
+        expected = pd.Series([scalar] * 3, index=[1, 2, 3], dtype=dtype)
+        self.assert_series_equal(result, expected)
+
+    def test_series_constructor_scalar_with_one_element_index(self, data):
```

Comment:
I would maybe just combine this one with the test above (both are about scalar with index) ?

(and one test less to override in the subclasses)